### DISCUSSION TITLE
Boss timers: Vardorvis

### DISCUSSION THREAD

**Discussion by GxH033** [runelite/runelite#17865] [Ideas]
Boss timers: Vardorvis

**Comment by Nightfirecat** [comment]
?

**Reply by GxH033** [reply]
There is no boss timer for when Vardorvis spawns

**Reply by Nightfirecat** [reply]
Vardorvis respawning does not initiate the fight, so no timer is started for him dying.

**Reply by GxH033** [reply]
Giant mole has a timer

**Reply by GxH033** [reply]
Doesn't Kraken have a timer as well
